app/src/lib/stores/simulation/domain/milestone.ts

```
@@ -0,0 +1,21 @@
+import type { UniqueObject } from "./base";
+
+export interface AchievingMethod {
+}
```

Biome (2.1.2)

[error] An empty interface is equivalent to {}.

Safe fix: Use a type alias instead.


(lint/suspicious/noEmptyInterface)